recipe/meta.yaml

```diff
@@ -38,10 +48,46 @@ test:
     - pydy.viz
     - pydy.viz.tests
 
+outputs:
+  - name: pydy-optional
+    version: {{ version }}
+    requirements:
+      run:
+        - python
+        - pydy {{ version }}
+        # Optional dependencies:
+        - notebook 4.*
+        - ipywidgets 4.*
+        - cython >=0.23.4
+        - theano >=0.8.0
```

Comment:
These would be enforced by the run_constrained above, so I could leave off the specifiers.